Vagrantfile

```diff
@@ -116,8 +116,25 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   SLAVES_COUNT.times do |i|
     index = i + 1
     ip_index = i + 3
+
     config.vm.define "solar-dev#{index}" do |config|
 
+    config.vm.provider :virtualbox do |v|
```

Comment:
This better to be done as a resource, please ask @loles for assistance. We have the same task for the cinder-volume VG